README.md

```diff
@@ -4,6 +4,8 @@ This extension is a [Firefox SHIELD](https://support.mozilla.org/en-US/kb/shield
 
 ### Build the extension
 
+Note: this step is optional, you can download a [recent version of the zip here (private URL)](https://drive.google.com/drive/folders/1sRQDC2brfsXGTcJO6ElkbeqakdKA1cln?usp=sharing).
```

Comment:
Since we're in a private repo, not sure if it'd be any easier to possibly use use the [GitHub releases](https://github.com/raymak/vpn-recommendation-shield-study/releases) to distribute ZIP files.

Reply:
@pdehaan we are not in a private repo. But I don't see a problem using GitHub releases regardless.